src/Microsoft.DotNet.Build.Tasks/PackageFiles/VersionTools.targets

```diff
@@ -29,6 +35,7 @@
     <UpdateDependencies DependencyBuildInfo="@(DependencyBuildInfo)"
                         ProjectJsonFiles="@(ProjectJsonFiles)"
                         XmlUpdateStep="@(XmlUpdateStep)"
+                        BuildInfoCacheDir="$(BuildInfoCacheDir)"
```

Comment:
Oh, you're right, I must not have noticed that this was specifically about UpdateDependencies. The cache is never used here, so I'll remove that input.